### ISSUE TITLE
[BUG] TypeError: Cannot set property readableObjectMode of #<Readable> which has only a getter

### ISSUE THREAD

**Issue by RAnders00** [b4nst/stream-mock#26]
**Description**

I'm trying to create a `DuplexMock` using `stream-mock@2.0.2` on node.js version `v12.4.0` (from my testing, `v11.15.0` seems to be unaffected), and it throws the following exception when calling the constructor:

```
project\node_modules\stream-mock\lib\duplex\DuplexMock.js:11
        this.readableObjectMode = options.objectMode || options.readableObjectMode;
                                ^

TypeError: Cannot set property readableObjectMode of #<Readable> which has only a getter
    at new DuplexMock (project\node_modules\?[4mstream-mock?[24m\lib\duplex\DuplexMock.js:11:33)
    at Object.<anonymous> (project\test.js:3:1)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:837:10)
    at internal/main/run_main_module.js:17:11?[39m
```

**To Reproduce**

```javascript
const { DuplexMock } = require('stream-mock');

new DuplexMock();
```

**Expected behavior**

Should create the `DuplexStream` normally.

**Environment**
- stream-mock version: 2.0.2
- node version: 12.4.0
 - OS: Windows 10

**Additional context**

**Comment by RAnders00** [comment]
I believe this node.js commit broke it: https://github.com/nodejs/node/commit/b4735ecebb390f467bba5b2d467f27f88dbbf09e

**Comment by b4nst** [comment]
Hey, thanks for reporting that. I will try to fix it asap.

**Comment by b4nst** [comment]
fixed in [v2.0.3](https://www.npmjs.com/package/stream-mock/v/2.0.3)
